CMake/External_VTK.cmake

```diff
@@ -54,13 +54,31 @@ set(vtk_cmake_args ${vtk_cmake_args}
   )
 
 # Qt
-add_package_dependency(
-  PACKAGE VTK
-  PACKAGE_DEPENDENCY Qt
-  PACKAGE_DEPENDENCY_ALIAS Qt4
-)
-if(QT_QMAKE_EXECUTABLE)
-  set(BUILD_QT_WEBKIT ${QT_QTWEBKIT_FOUND})
+if(Qt_version_major EQUAL 4)
+  add_package_dependency(
+    PACKAGE VTK
+    PACKAGE_DEPENDENCY Qt
+    PACKAGE_DEPENDENCY_ALIAS Qt4
+  )
+  if(QT_QMAKE_EXECUTABLE)
```

Comment:
You'd have to explicitly change VTK's Qt version to 4 via:

```cmake
-DVTK_QT_VERSION:STRING=4
```

Reply:
Ack, I missed that we aren't already passing that...